### PR TITLE
Fetch Country list and display in a dropdown

### PR DESCRIPTION
**Pull request by adenzil**
- Fetch list of countries from Holiday API
- display countries in a dropdown

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -51,3 +58,13 @@ export default function Home() {
     </div>
   )
 }
+
+Home.getInitialProps = async function() {
```
Comment (neilmalgaonkar) [inline pages/index.js:62]: This is not wrong but you have to implement the frontend only solution for first phase

Comment (adenzil) [reply pages/index.js:62]: How is the front-end only solution different ? example ?

Comment (neilmalgaonkar) [reply pages/index.js:62]: This code is executed on the server side when your code is compiled. This does not execute on the browser. You can try it with adding console to it.
I want the implementation which will work on Browser only and no server side rendering.

Comment (neilmalgaonkar) [reply pages/index.js:62]: https://www.google.com/search?q=react+fetch+data&rlz=1C5CHFA_enIN741IN741&oq=react+fetch+data&aqs=chrome.0.69i59j0l6j69i60.3230j0j7&sourceid=chrome&ie=UTF-8
https://reactjs.org/docs/faq-ajax.html

File: pages/index.js [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+    const savedCountries = localStorage.getItem("countries");
+    if (savedCountries) {
+      setCountries(JSON.parse(savedCountries));
+    } else {
+      // To fetch .env variables from next.config.js
+      const {publicRuntimeConfig} = getConfig()
+      const {holidayApiURL, holidayApiKey} = publicRuntimeConfig
+
+      fetch(holidayApiURL + 'countries?key=' + holidayApiKey)
+      .then(res => res.json())
```
Comment (neilmalgaonkar) [inline pages/index.js:20]: Indentation

**Pull request merged**